code/src/CSS/Detail.css

```diff
@@ -0,0 +1,60 @@
+
+.detailPage {
+    position: relative;
+}
+
+.detailPage .background {
+    position: absolute;
+    background-size: cover;
+    background-repeat: no-repeat;
+    min-height: 100vh;
+    display: flex;
+    flex-direction: column;
+    align-content: flex-start;
+    justify-content: flex-end;
+}
```

Comment:
Nice way to set the css for multiple elements.

We saw that you have a black background showing in the right side of your page in desktop. We fixed that by adding this line of code: 
"min-width: 100vw;"

1. That made the poster image take over the side completely. But by deleting your line 40 - it works. Try it out and see what you think about it!